src/main/resources/application.conf

```diff
@@ -38,110 +46,221 @@ etc-client {
   }
 
   mining {
+    # Maximum number of ommers kept in the pool
     ommers-pool-size = 30
+
+    # Block cache size
+    # TODO: explain better, what is this cache?
     block-cashe-size = 30
+
+    # Miner's coinbase address
     coinbase = "0011223344556677889900112233445566778899"
-    pooling-services-timeout = 5.seconds
+
+    ommer-pool-query-timeout = 5.seconds
   }
 
   blockchain {
+    # Frontier block number
     frontier-block-number = "0"
+
+    # Homestead fork block number
+    # Doc: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2.md
     homestead-block-number = "1150000"
+
+    # EIP-150 fork block number
+    # Doc: https://github.com/ethereum/EIPs/issues/150
     eip150-block-number = "2500000"
+
+    # EIP-160 fork block number
+    # Doc: https://github.com/ethereum/EIPs/issues/160
     eip160-block-number = "3000000"
+
+    # Difficulty bomb pause block number
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1010.md
     difficulty-bomb-pause-block-number = "3000000"
+
+    # Difficulty bomb continuation block number
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1010.md
     difficulty-bomb-continue-block-number = "5000000"
 
-    // Doc: https://blog.ethereum.org/2016/07/20/hard-fork-completed/
+    # DAO fork block number (Ethereum HF/Classic split)
+    # https://blog.ethereum.org/2016/07/20/hard-fork-completed/
     dao-fork-block-number = "1920000"
-    dao-fork-block-total-difficulty = "39490964433395682584"
+
+    # The hash of the accepted DAO fork block
     dao-fork-block-hash = "94365e3a8c0b35089c1d1195081fe7489b528a84b22199c916180db8b28ade7f"
 
+    # Starting nonce a an empty account. Some networks (like Morden) use different values.
     account-start-nonce = "0"
 
+    # The ID of the accepted chain
     chain-id = "3d"
 
+    # Custom genesis JSON file
     custom-genesis-file = null
 
-    # ECIP-1017
+    # Monetary policy parameters
+    # Doc: https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1017.md
     monetary-policy {
+      # Block reward in the first era
       first-era-block-reward = "5000000000000000000"
+
+      # Monetary policy era duration in number of blocks
       era-duration = 5000000
+
+      # Rate at which rewards get reduced in successive eras.
+      # Value in range [0.0, 1.0]
       reward-reduction-rate = 0.2
     }
   }
 
-  fast-sync {
+  sync {
+    # Whether to enable fast-sync
     do-fast-sync = true
+
+    # TODO: explain
     peers-scan-interval = 3.seconds
+
+    # Duration for blacklisting a peer
+    # TODO: explain better, reasons for blacklisting?
     blacklist-duration = 30.seconds
+
+    # TODO: explain
     start-retry-interval = 5.seconds
+
+    # TODO: explain
     sync-retry-interval = 5.seconds
+
+    # TODO: explain
     peer-response-timeout = 10.seconds
+
+    # TODO: explain
     print-status-interval = 2.seconds
+
+    # TODO: explain
```

Comment:
How often to dump fast-sync status to disk. If app is restarted, fast-sync will continue from this point.